core/wiki/macros/colour-picker.tid

```diff
@@ -1,58 +1,57 @@
 title: $:/core/macros/colour-picker
 tags: $:/tags/Macro
 
-\define colour-picker-update-recent()
+\procedure colour-picker-update-recent()
 <$action-listops
 	$tiddler="$:/config/ColourPicker/Recent"
-	$subfilter="$(colour-picker-value)$ [list[$:/config/ColourPicker/Recent]remove[$(colour-picker-value)$]] +[limit[8]]"
+	$subfilter="[<colour-picker-value>] [list[$:/config/ColourPicker/Recent]remove<colour-picker-value>] +[limit[8]]"
 />
 \end
 
-\define colour-picker-inner(actions)
-<$button tag="a" tooltip="""$(colour-picker-value)$""">
-$(colour-picker-update-recent)$
-<$transclude $variable="__actions__"/>
-
-<span style="display:inline-block; background-color: $(colour-picker-value)$; width: 100%; height: 100%; border-radius: 50%;"/>
-
+\procedure colour-picker-inner(actions)
+<$button tag="a" tooltip=<<colour-picker-value>>>
+<<colour-picker-update-recent>>
+<$transclude $variable="actions"/>
+<span style.display="inline-block" style.backgroundColor=<<colour-picker-value>> style.width="100%" style.height="100%" style.borderRadius="50%"/>
 </$button>
 \end
 
-\define colour-picker-recent-inner(actions)
+\procedure colour-picker-recent-inner(actions)
 \whitespace trim
-<$set name="colour-picker-value" value="$(recentColour)$">
-<$macrocall $name="colour-picker-inner" actions=<<__actions__>>/>
```

Comment:
@Jermolene -- Do we really need to convert `<$macrocall` to a `<$transclude $variable` -- I personally like the macrocall much more, since it is a 100 times more verbose and human readable than the transclusion?

Reply:
For me the reason is that macrocall widget is [marked deprecated](https://tiddlywiki.com/prerelease/#MacroCallWidget) since v5.3.0.

Reply:
I also support $transclude. This makes learning easier.
Also procedures replace macros, so $macrocall for procedures is a bit confusing. All in all $transclude is my favorite to go.

Reply:
Yes I think we have decided to deprecate the `<$macrocall>` widget. I am not against trying evolving the shortcut syntax. It would be very nice to be able to just do something like `<<colour-picker-inner actions=<<actions>>>>`.